lib/bolt/outputter/human.rb

```diff
@@ -93,6 +93,21 @@ def print_table(results)
         )
       end
 
+      # @param [Hash] A hash representing the task
+      def print_task(task)
```

Comment:
I think its less than ideal that `print_task` prints the metadata while `print_plan` prints the results of a plan.
it should either be` print_task_info` or `print_plan_results` probably